### ISSUE TITLE
Mobile app does not update regions

### ISSUE THREAD

**Issue by marceloscarone** [fieldsight/fieldsight-mobile#526]
#### Software and hardware versions 
FieldSight 3.3.3, Android 9, huawei mate 10 pro

#### Problem description
When users are assigned a region from the web application, after synchronization, the region does not appear in the mobile application. You need to log out and log in for the region to appear.
![Screenshot_20200218_100642](https://user-images.githubusercontent.com/50563569/74738858-84ec1f00-5236-11ea-8a8a-f8e0d305398b.jpg)
![Screenshot_20200218_100446_org bcss collect android](https://user-images.githubusercontent.com/50563569/74738872-86b5e280-5236-11ea-841b-6adf05e6a4ad.jpg)
![Captura](https://user-images.githubusercontent.com/50563569/74738878-87e70f80-5236-11ea-8ebb-dbc1963e688a.PNG)
![Screenshot_20200218_095352_org bcss collect android](https://user-images.githubusercontent.com/50563569/74738881-887fa600-5236-11ea-8877-003d405914ab.jpg)

**Comment by yubarajpoudel** [comment]
@marceloscarone , did you sync the project again after the regions are assigned from the web? App content does not update itself, instead, you have to sync again to see the updates.

**Comment by marceloscarone** [comment]
Yes, I updated the mobile application after assigning a new region from the web application, and I can't find the region. Only by logging out and starting again, I have gained access to the region.
If you need, this is my user: marcelos@unops.org
